doc/ModulesDevelopment.md

```diff
@@ -328,9 +328,77 @@ For a new feature, just create a new branch for it and use the same procedure. D
 
 General ALICE Git guidelines can be accessed [here](https://alisw.github.io/git-tutorial/).
 
-## Raw data source
+## Data sources
```

Comment:
please fix the references to "Raw data source"

Reply:
Fixed the table of contents.